fw-child/template/header-fixed.php

```diff
@@ -1,4 +1,4 @@
-<div id="floating-header" class="fixed-top col-1">
+<div id="floating-header" class="fixed-top col-2 col-md-1">
```

Comment:
@xfrenette-crim this code in particular is here to match the sidebar that can be found in some of the internal pages.

![Screenshot_20240621_152806](https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/assets/169822416/88c05448-b455-43f5-9cfa-6034c8351117)

This classes can't be removed because it would span the entire width.

![Screenshot_20240621_152519](https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/assets/169822416/c961e562-ec29-4e00-9834-c2257e89dcfe)

Unfortunately as a side effect, it affects the logo width, because it is inside of this container. I pushed a few more changes in css to try to achieve what you suggest, but in the end, in most of the cases, the width is constrained by the column width.

Reply:
Ok. We will setup a new task to fix this problem of the sidebar and the logo.